### PR TITLE
GH-91 Refactor gradle module setup.

### PR DESCRIPTION
**Pull request by vLuckyyy**

### CODE REVIEW

**Review by CelestialViolet — CHANGES_REQUESTED** [review]
https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html

**Comment by CelestialViolet** [comment]
sigma-male-sigma

File: buildSrc/src/main/kotlin/eternalcode.java.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+plugins {
+    `java-library`
+}
+
+group = "com.eternalcode"
+version = "1.0.7"
+
+repositories {
+    mavenCentral()
+    mavenLocal()
```
Comment (Rollczi) [inline buildSrc/src/main/kotlin/eternalcode.java.gradle.kts:10]:
```suggestion
```

File: core/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.10.0")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.10.0")
+}
+
+tasks {
+    runServer {
+        minecraftVersion("1.19.3")
+    }
+
+    getByName<Test>("test") {
```
Comment (Rollczi) [inline core/build.gradle.kts:63]:
```suggestion
    withType<Test> {
```

**Review by Rollczi — APPROVED** [review]

**Pull request merged**